lnwallet/channel.go

```diff
@@ -2704,6 +2704,14 @@ func (v *HtlcView) AuxTheirUpdates() []AuxHtlcDescriptor {
 	return fn.Map(v.Updates.Remote, newAuxHtlcDescriptor)
 }
 
+// FetchLatestHTLCView returns the latest HTLC view of the lightning channel.
+// The latest log indices are used for both the local and remote update logs.
+func (lc *LightningChannel) FetchLatestHTLCView() *HtlcView {
+	return lc.fetchHTLCView(
+		lc.updateLogs.Remote.logIndex, lc.updateLogs.Local.logIndex,
```

Comment:
I think we also need to set the `NextHeight` on the `HtlcView` the same way we do here: https://github.com/lightningnetwork/lnd/blob/7e60d50bcad78a80a3803315347094dfe009d5f7/lnwallet/channel.go#L2771
Which then means we're accessing either the local or remote commit chain. So we should either put "local" in the name of this method or add the `whoseCommitChain lntypes.ChannelParty` as a parameter.

Reply:
Hmm, or maybe the `ComputeView` in `tapd` only currently works because the `NextHeight` is zero?
Either way, there are too many uncertainties IMO and it seems too easy to just work... So I think this requires a bit more investigation to make sure we don't implement something that is incorrect or only correct in some cases.